plugins/Referrers/tests/System/ApiTest.php

```diff
@@ -63,7 +63,7 @@ public function getApiForTesting()
                 'date' => '2010-01-01',
                 'periods' => 'year',
                 'testSuffix' => 'phpSerialized',
-                'format' => 'original',
+                'format' => 'php',
```

Comment:
@katebutler as discussed we probably need to adjust instead the original handler to respect `hideIdSubDatatable`. The test was added to ensure when the dataTable object is serialized that no random information is being added see https://github.com/matomo-org/matomo/commit/46ed8c2eef4990688c1188b31bc8d8229087daab  . But now an array is serialized and the actual logic is no longer tested. If possible be good to still use `original` and also add a comment that this test is testing the serialized DataTable